qtgui/settingutil.h

```diff
@@ -0,0 +1,23 @@
+#ifndef SETTINGUTIL_H
+#define SETTINGUTIL_H
+#include <QSettings>
+
+//TODO: сделать ui для настройки
+namespace andeme {
+class settingutil{
+public:
+    settingutil();
+    ~settingutil(){WriteSettings();}
+    void ReadSettings();
+    void WriteSettings();
+    void SetDefaultSettings();
+    std::string getHostname();
+    std::string getUsername();
+private:
+    QSettings m_settings;
+    QString m_hostname;
+    QString m_username;
```

Comment:
Давай придерживаться Google Code Style (https://google.github.io/styleguide/cppguide.html#Variable_Names)
```suggestion
    QString username_;
```
Аналогично для остального